app/assets/onepager/js/receive.js

```diff
@@ -8,4 +33,114 @@ window.onload = function() {
     }
     $('#network').val(document.web3network);
   });
-};
\ No newline at end of file
+};
+
+$(document).ready(function() {
+  $('#receive').click(function(e) {
```

Comment:
Can we replace this with `$(document).on('click', '#receive', function(e) ` since `click()` function has been deprecated by the jQuery team?

Reply:
yes